dist/onfido.min.js

```diff
@@ -1,4 +1,4 @@
var r=n(10),o=n.n(r),i=n(132),a=n.n(i),s=(n(22),n(0)),c=n(313),u=n(15),l=n(31),p=n.n(l),f=n(35),d=n.n(f),h=n(314),m=n.n(h),v=n(250),g=n.n(v),b=n(315),y=n(316),_=n(317),k=n(318),w=n(24),O={en:b,es:y},x={en:_,es:k},C=function(e,t,n,r){return t.locale(e),t.extend(n),u.g||t.extend(r),t},S=function(e,t){!function(e,t){var n=new m.a(t),r=d()(e).call(e,function(e){return!n.has(e)});r.length&&console.warn("Missing keys:",r)}(p()(t.phrases),function(e){return p()(new g.a({phrases:e}).phrases)}(e))},j=function(e,t){if(O[e])return C(e,t,O[e],x[e]);console.warn("Locale not supported")},E=function(e,t){return"string"==typeof e?j(e,t):function(e,t){S(e.phrases,t);var n=j(e.locale,t)||t;return C(e.locale,n,e.phrases,e.mobilePhrases)}(e,t)},T=Object(w.c)(function(e){var t=function(){var e=new g.a({onMissingKey:function(){return null}});return C("en",e,O.en,x.en)}();return e&&E(e,t)||t});n.d(t,"a",function(){return N}),n.d(t,"b",function(){return A});var P=Object(c.createContext)({}),N=function(e){var t,n=e.language,r=e.children,o=T(n),i=a()(t=o.t).call(t,o);return Object(s.h)(P.Provider,{value:{language:o.currentLocale,translate:i,parseTranslatedTags:function(e,t){return Object(u.h)(i(e),t)}}},r)},A=function(e){return function(t){return Object(s.h)(P.Consumer,null,function(n){return Object(s.h)(e,o()({},t,n))})}}},function(e,t,n){var r=n(125);function o(e,t){for(var n=0;n<t.length;n++){var o=t[n];o.enumerable=o.enumerable||!1,o.configurable=!0,"value"in o&&(o.writable=!0),r(e,o.key,o)}}e.exports=function(e,t,n){return t&&o(e.prototype,t),n&&o(e,n),e}},function(e,t,n){var r=n(358);function o(){return e.exports=o=r||function(e){for(var t=1;t<arguments.length;t++){var n=arguments[t];for(var r in n)Object.prototype.hasOwnProperty.call(n,r)&&(e[r]=n[r])}return e},o.apply(this,arguments)}e.exports=o},function(e,t,n){e.exports={"absolute-center":"onfido-sdk-ui-Theme-absolute-center",step:"onfido-sdk-ui-Theme-step",fullScreenStep:"onfido-sdk-ui-Theme-fullScreenStep",fullHeightContainer:"onfido-sdk-ui-Theme-fullHeightContainer",fullHeightMobileContainer:"onfido-sdk-ui-Theme-fullHeightMobileContainer",navigationBar:"onfido-sdk-ui-Theme-navigationBar",content:"onfido-sdk-ui-Theme-content",scrollableContent:"onfido-sdk-ui-Theme-scrollableContent",fullScreenContentWrapper:"onfido-sdk-ui-Theme-fullScreenContentWrapper",footer:"onfido-sdk-ui-Theme-footer",thickWrapper:"onfido-sdk-ui-Theme-thickWrapper","mbottom-large":"onfido-sdk-ui-Theme-mbottom-large",center:"onfido-sdk-ui-Theme-center",icon:"onfido-sdk-ui-Theme-icon",help:"onfido-sdk-ui-Theme-help",header:"onfido-sdk-ui-Theme-header",helpList:"onfido-sdk-ui-Theme-helpList"}},function(e,t,n){var r=n(165),o=n(171),i=n(175),a=n(179),s=n(141),c=n(142),u=n(1);e.exports=function(e){for(var t=1;t<arguments.length;t++){var n,l=null!=arguments[t]?arguments[t]:{},p=c(l);"function"==typeof s&&(p=a(p).call(p,i(n=s(l)).call(n,function(e){return o(l,e).enumerable}))),r(p).call(p,function(t){u(e,t,l[t])})}return e}},function(e,t,n){var r;
```

Comment:
we should only update the dist for release, if possible it would be good to remove this diff

Reply:
Have rolled back the changed minified file so it matches what's on `development` now